src/Streamer.h

```diff
@@ -18,6 +18,7 @@
 
 #include "KafkaW/KafkaW.h"
 
+#include <chrono>
```

Comment:
Was this intentional?

Reply:
Yes, this was intentional. I have not looked at the "include paths" but when I removed Alloc.h, some of the build nodes complained about missing std::chronological. With that said, you do have a point in that those include-statements could (should) probably be in the cpp-file.